nat/nat.go

```diff
@@ -116,6 +118,18 @@ func (dev *Mapper) keepPortMapping(protocol string, intPort, extPort uint16, des
 				dev.log.Warn("Renew NAT Traversal failed from external port %d to internal port %d with %s",
 					extPort, intPort, err)
 			}
+			if ip != nil {
+				newIp, err := dev.r.ExternalIP()
+				if err != nil {
+					oldIp := ip.Ip().IP
+					ip.UpdateIP(newIp)
+					if !oldIp.Equal(newIp) {
+						dev.log.Info("Renew ExternalIP updated to %s", newIp)
```

Comment:
Minor nit: Can we change this to `"External IP updated to: %s"` ? I think that's a little more clear